thinkhazard/static/less/report.less

```diff
@@ -238,6 +238,11 @@ footer {
   }
 }
 
+.logo {  
+  font-size: 25.2px;
```

Comment:
Even if written in the document, this value looks weird to me. Please remove it.